### ISSUE TITLE
v0.4.0 - cannot start simulation (GUI)

### ISSUE THREAD

**Issue by yjkm** [NREL/bifacial_radiance#379]
Good morning, 

It's me again!

I noticed that the v0.4.0 was released a week ago - so I decided to check it out. Alas, once I downloaded it, I opened up the GUI and tried to run a simple simulation and got the following error. 

```
runfile('C:/Users/yjkm/Documents/bifacial_radiance-main/bifacial_radiance/gui.py', wdir='C:/Users/yjkm/Documents/bifacial_radiance-main/bifacial_radiance')
Read all values

New bifacial_radiance simulation starting. 
Version:  0+unknown
path = C:\Users\yjkm\Documents\bifacial_radiance-main\bifacial_radiance\TEMP
analysisParamsDict['modWanted'] set to middle module by default
analysisParamsDict['rowWanted'] set to middle row by default
Getting weather file: USA_AZ_Safford.AWOS.722747_TMY3.epw
OK!
Reading weather file EPWs\USA_AZ_Safford.AWOS.722747_TMY3.epw
8760 line in WeatherFile. Assuming this is a standard hourly WeatherFile for the year for purposes of saving Gencumulativesky temporary weather files in EPW folder.
Coercing year to 2021
Saving file EPWs\metdata_temp.csv, # points: 8760
Calculating Sun position for Metdata that is right-labeled  with a delta of -30 mins. i.e. 12 is 11:30 sunpos
Loading albedo, 1 value(s), 0.620 avg
1 nonzero albedo values.
Available module names: ['PrismSolar-Bi60', 'Prism_Solar_Bi60', 'basic-module', 'test-module']

Module Name: Prism_Solar_Bi60
Module Prism_Solar_Bi60 updated in module.json
Pre-existing .rad file objects\Prism_Solar_Bi60.rad will be overwritten

Error: module name Prism Solar Bi60 doesnt exist
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\yjkm\Anaconda3\lib\tkinter\__init__.py", line 1892, in __call__
    return self.func(*args)
  File "C:\Users\yjkm\Documents\bifacial_radiance-main\bifacial_radiance\gui.py", line 390, in runBifacialRadiance
    self.data, self.analysis = bifacial_radiance.modelchain.runModelChain(simulationParamsDict=simulationParamsDict,
  File "c:\users\yjkm\documents\bifacial_radiance-main\bifacial_radiance\modelchain.py", line 140, in runModelChain
    scene = demo.makeScene(module=simulationParamsDict['moduletype'],
  File "c:\users\yjkm\documents\bifacial_radiance-main\bifacial_radiance\main.py", line 2110, in makeScene
    self.scene = SceneObj(module)
  File "c:\users\yjkm\documents\bifacial_radiance-main\bifacial_radiance\main.py", line 2811, in __init__
    self.module = ModuleObj(name=module)
  File "c:\users\yjkm\documents\bifacial_radiance-main\bifacial_radiance\module.py", line 110, in __init__
    self._saveModule(savedata=None, json=False,
  File "c:\users\yjkm\documents\bifacial_radiance-main\bifacial_radiance\module.py", line 276, in _saveModule
    if rewriteModulefile & os.path.isfile(self.modulefile):
AttributeError: 'ModuleObj' object has no attribute 'modulefile'
```

I do not know what went wrong to be quite honest, any help would be much appreciated.  I mean I can downgrade the PVLIB library and bifacial_radiance but I do not know how to do that for the latter. 

![image](https://user-images.githubusercontent.com/98945475/153364918-6036b2fa-1e8d-4783-941f-7b204063eb93.png)

**Comment by kush987** [comment]
Select the second option of module parameters list

**Comment by cdeline** [comment]
This is fixed in the main branch if you're able to install directly from Github.  If not we should be able to get a development pre-release out shortly once we've squashed all of the other bugs.

**Comment by cdeline** [comment]
Fixed in v0.4.1
